harvester/harvester-installer
# feat: bump RKE2 to v1.28.12+rke2r1 and Rancher to v2.8.5

**Solution:**

Bump RKE2 to v1.28.12+rke2r1 and Rancher to v2.8.5.

**Related Issue:**
https://github.com/harvester/harvester/issues/6262

**Test plan:**
1. Create harvester in airgap environment and make sure all pod can up.
2. Check basic VM operation can work.
